test/unit/com/unifina/signalpath/PropagatorSpec.groovy

```diff
@@ -3,11 +3,14 @@ package com.unifina.signalpath
 import com.unifina.ModuleTestingSpecification
 import com.unifina.data.FeedEvent
 import com.unifina.datasource.DataSource
+import com.unifina.domain.security.SecUser
 import com.unifina.feed.MasterClock
 import com.unifina.signalpath.simplemath.Count
 import com.unifina.signalpath.time.ClockModule
 import com.unifina.utils.Globals
+import grails.test.mixin.Mock
 
+@Mock(SecUser)
```

Comment:
Yep the change in `ModuleTestingSpecification` adds this pollution. Please revert that change and the added `@Mock`s to unrelated files